Dockerfile.mysql

```diff
@@ -7,6 +7,7 @@ ENV MYSQL_DATABASE=app_db
 ENV MYSQL_USER=app_user
 ENV MYSQL_PASSWORD=1234
```

Comment:
It's not secure to hardcode passwords in the Dockerfile. Consider using Docker secrets or environment variables passed at runtime to manage sensitive information like passwords.